tensorboard/data/provider.py

```diff
@@ -109,11 +109,14 @@ class DataProvider(metaclass=abc.ABCMeta):
     execution.
 
     Each run within an experiment may specify its own value for a
-    hyperparameter. Runs that were logically executed together with the same set
-    of hyperparameter values form a hyperparameter session group. When querying
-    for hyperparameter values for a set of experiments, the result will group
-    runs by hyperparameter session group and provide one set of hyperparameter
-    values for each group.
+    hyperparameter. Runs that were logically executed together with the same
```

Comment:
I would insert an additional sentence that first introduces the name of "session group" as I think helps to show a distinction between "session" and "session group" early. I would also perhaps quote the terms as they are being introduced.

Runs that were logically executed together with the same set of hyperparameter values form a hyperparameter `session`. Sessions that include the same hyperparameter values can be grouped together in a hyperparameter `session group`. Often a session...

Reply:
Done.